CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+Possible log types are `[add]`, `[bug]`, `[change]` and `[info]`.
+
+
+### v0.2.0 (2015-11-22)
+
+- [add] Custom `Decodable` implementation for `Optional` (#12)
```

Comment:
I think the major change is *decoding* support. The custom implementation was just needed for that. Previously decoding was not possible at all, was it?